includes/admin/class-wc-rest-stripe-settings-controller.php

```
@@ -610,6 +612,10 @@ private function update_oc_settings( WP_REST_Request $request ) {
 			}
 			$current_value = $this->gateway->get_option( $attribute );
```

PHPMD (2.15.0)

[error] The variable $current_value is not named in camelCase. (undefined)

(CamelCaseVariableName)